packages/gatsby-graphiql-explorer/src/app/webpack.config.js

```diff
@@ -69,4 +69,16 @@ module.exports = {
   stats: {
     warnings: false,
   },
+  // below are temporary aliases that link to not yet published upcoming graphiql feature ( https://github.com/graphql/graphiql/pull/1750 )
+  resolve: {
+    alias: {
+      graphiql: `/Users/misiek/dev/graphiql/packages/graphiql`,
+      "codemirror-graphql": `/Users/misiek/dev/graphiql/packages/codemirror-graphql`,
+      "graphql-language-service-interface": `/Users/misiek/dev/graphiql/packages/graphql-language-service-interface`,
+      "graphql-language-service": `/Users/misiek/dev/graphiql/packages/graphql-language-service`,
```

Comment:
If you want to try locally (before new graphiql is released) - you need to clone `graphiql` monorepo ( https://github.com/graphql/graphiql ), build packages and adjust those paths (those are mine local paths), that made it a bit easier to work with than using something like `npm/yarn link` etc